### ISSUE TITLE
Just curious if you ever had some code written for this project

### ISSUE THREAD

**Issue by Tiberriver256** [daviwil/PSOrgMode#1]

**Comment by daviwil** [comment]
Hey Micah!  Honestly, I don't remember where I got with this.  Are you using org-mode in Emacs?

**Comment by Tiberriver256** [comment]
Yeah, just getting started with it and kind of kicking the tires on it a little bit. I'm having trouble deciding between it and OneNote. Org-mode is way more flexible but OneNote has "Send-To" built into pretty much everything.

I had some interesting ideas on syncing todo's from Outlook and tasks from MSProject and saw your repo here so I thought I would ask. Don't go crazy on anything for my whims on productivity though :)
